src/pages/rules/Rules.tsx

```diff
@@ -0,0 +1,64 @@
+import "./Rules.css";
+import { useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { useQuiz } from "../../context/quiz-context";
+import { getQuiz } from "../../services/getDataService";
+import { Quiz } from "../pages";
+
+export const Rules = (): JSX.Element => {
+  const { categoryId } = useParams();
+  const {
+    quizState: { activeQuestion, points },
+    quizDispatch,
+  } = useQuiz();
+
+  useEffect(() => {
+    (async () => {
+      const res = await getQuiz(categoryId || "");
```

Comment:
destructure { quizName, questions, points }

Reply:
it's type is causing issue while destructuring